kamon-core/src/main/scala/kamon/util/executors/ExecutorMetricRecorder.scala

```diff
@@ -54,6 +44,17 @@ object ForkJoinPools {
   }
 }
 
+trait ForkJoinPoolsImplicitsLowPrio {
+  implicit object ScalaForkJoin extends ForkJoinMetrics[ForkJoinPool] {
```

Comment:
`ForkJoinPool` is deprecated and now justa a `JavaForkJoinPool` so this causes implicit ambiguity.  Moving this to a lower priority implicit should work in both cases.